score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java

```diff
@@ -519,6 +543,14 @@ public void addRoiValue(Double roiValue) {
         contextMap.put(ExecutionParametersConsts.EXECUTION_TOTAL_ROI, currentRoiValue + roiValue);
     }
 
+    public StatefulSessionStack getStatefulSessionStack() {
```

Comment:
should be consistent when naming get method and se method. Use setStatefulSessionsStack and getStatefulSessionsStack